README.md

```diff
@@ -483,7 +483,7 @@ The default values for each rule are described below.
 | no_property_description     | warning |
 | description_mentions_json   | warning |
 | array_of_arrays             | warning |
-| inconsistent_property_type  | warning |
+| inconsistent_property_type  | warning, code, default, type, value] |
```

Comment:
For rules with config options like `warning, lower_snake_case` and `case_convention` in the name, it's fairly obvious what the configuration represents.

In this new case, it may not be as obvious. I think we should document what the list means. Something like

```suggestion
| inconsistent_property_type  | warning, [code, default, type, value] | (List of property names excluded for this check)
```

I don't know if that would screw up the table formatting though. Maybe there's another place we could document it.

Reply:
I did need to modify this slightly to get the table to render properly